pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/metrics/PrometheusMetricsProvider.java

```diff
@@ -107,20 +74,30 @@ public void stop() {
 
     @Override
     public StatsLogger getStatsLogger(String scope) {
-        return this.cachingStatsProvider.getStatsLogger(scope);
+        Map<String, String> labels = new HashMap<>();
```

Comment:
@hangc0276 Previously there was a caching layer called `cachingStatsProvider` which saved in a map every `StatsLogger` created via `getStatsLogger`. If it was created previously it would return it. 
Now it is removed